types/lru-cache/index.d.ts

```diff
@@ -52,112 +53,109 @@ declare class LRUCache<K, V> implements Iterable<[K, V]> {
      * as in `cache.set(key, undefined)`. Use `cache.has()` to determine
      * whether a key is present in the cache at all.
      */
-    get(key: K, options?: LRUCache.GetOptions): V | undefined;
+    public get<T = V>(key: K, options?: LRUCache.GetOptions): T | undefined;
```

Comment:
The cache can contain different types, so by using a generic here we can let the developer be more specific about which type they will get back (optionally, if they know what they are doing).